### DISCUSSION TITLE
How I can change mantine-Group-root flex-wrap property in RadioGroup?

### DISCUSSION THREAD

**Discussion by phuriwatcmk** [mantinedev/mantine#2265] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/49318726/187082156-c0e844ff-4e73-43bb-93bb-e4c174933960.png)

In RadioGroup I need to change mantine-Group-root    flex-wrap: wrap; to flex-wrap: nowrap; ,I was tried change CSS property in  Radio component Styles API But it not works. There are any way to change it ?

![image](https://user-images.githubusercontent.com/49318726/187082486-6dfd5a96-81ed-4611-9619-4738c50c0005.png)

**Comment by rtivital** [comment, marked as answer]
`<Radio.Group sx={{ '& .mantine-Group-root': { flexWrap: 'nowrap' } }} />`
